lib/commands/sdk.js

```diff
@@ -446,6 +446,11 @@ SdkSubcommands.select = {
 					logger.error(__('Invalid Titanium SDK "%s"', value));
 					return false;
 				}
+				if (appc.version.lt(value, '3.2.0')) {
+					logger.log('');
+					logger.warn(__('Selected SDK contains deprecated environment detection code.'));
+					logger.warn(__('Please choose a 3.2 or newer SDK for more accurate information.'));
```

Comment:
The warning here needs to change. We shouldn't concern the user with detection code and such. We simply need to say:

```
Titanium SDK {VERSION} has been deprecated and will not work with future releases.
Please use Titanium SDK 3.2 or newer.
```